src/test/java/com/sap/oss/phosphor/fosstars/model/score/oss/OssArtifactSecurityScoreTest.java

```diff
@@ -130,7 +130,7 @@ public void testCalculateWithV200UsedAndOldVulnerability() {
         ARTIFACT_VERSION.value(new ArtifactVersion("2.0.0", LocalDateTime.now())));
 
     ScoreValue scoreValue = score.calculate(values);
-    assertTrue(DoubleInterval.closed(6, 7).contains(scoreValue.get()));
+    assertTrue(DoubleInterval.closed(4, 5).contains(scoreValue.get()));
```

Comment:
Looks like removal of LGTM feature costs 1 rating sub score, it is around 0.2 reduction in overall rating value like you showed earlier right?